Makefile

```diff
@@ -253,13 +253,14 @@ endif
 .PHONY: web-pre-process
 web-pre-process:
 	@echo ">> running documentation website pre processing"
-	@bash scripts/websitepreprocess.sh
+	@bash scripts/website/websitepreprocess.sh
```

Comment:
Why do we need to explicitly call bash? Rather than use the `#!` at the top of the script?

Reply:
Yeah...you're right. would fix up ASAP